mne_connectivity/spectral/epochs_multivariate.py

```diff
@@ -170,10 +177,11 @@ def reshape_csd(self):
 
 
 class _MultivariateCohEstBase(_EpochMeanMultivariateConEstBase):
-    """Base estimator for multivariate imag. part of coherency methods.
+    """Base estimator for multivariate coherency methods.
 
-    See Ewald et al. (2012). NeuroImage. DOI: 10.1016/j.neuroimage.2011.11.084
-    for equation references.
+    See:
+    - Ewald et al. (2012). NeuroImage. DOI: 10.1016/j.neuroimage.2011.11.084
+    - Vidaurre et al. (2019). NeuroImage. DOI: 10.1016/j.neuroimage.2019.116009
```

Comment:
I guess Ewald is for the real part and Vidaurre is for the imaginary part?

Can we add some short description here to describe what the method is computing?

Reply:
Ewald _et al._ (2012) is for the multivariate methods based on the imaginary part of coherency already implemented (MIC & MIM).

Vidaurre _et al._ (2019) is for the multivariate method based on the (complex) coherency (CaCoh). Here, the complex value is returned, but just like for coherency you can take the absolute part to get multivariate coherence if you're not interested in the phase angle of coupling.

@orabe can you add such a description when you have time?

Reply:
Done

https://github.com/mne-tools/mne-connectivity/blob/45458e94b54f16e18914ca8ba1a7e692b6989220/mne_connectivity/spectral/epochs_multivariate.py#L182-L187